backend/composer/models.py

```diff
@@ -240,18 +241,66 @@ class Meta:
         verbose_name_plural = "Anatomical Entities"
 
 
-class Layer(AnatomicalEntityMeta):
-    ...
+class Layer(models.Model):
+    layer_id = models.BigAutoField(primary_key=True, auto_created=True)
+    ae_meta = models.ForeignKey(AnatomicalEntityMeta, on_delete=models.CASCADE, related_name='layer_meta', null=True)
+
+    def __str__(self):
+        return self.ae_meta.name
+    
+    def clean(self):
+        if Layer.objects.filter(ae_meta=self.ae_meta).exists():
+            raise ValidationError('Layer with this ontology_uri already exists')
+        if Region.objects.filter(ae_meta=self.ae_meta).exists():
+            raise ValidationError('Region with this ontology_uri already exists')
+
+
+    def save(self, *args, **kwargs):
+        if Layer.objects.filter(ae_meta=self.ae_meta).exists():
+            raise ValueError('Layer with this Anatomical Entity Meta already exists')
+        elif Region.objects.filter(ae_meta=self.ae_meta).exists():
```

Comment:
issue (blocking): I believe we should allow a Layer to be created even if a Region already exists for the same anatomical entity meta. We only want to avoid duplicates within each table, this means no Layers repeated, no Regions repeated and no anatomical entities meta repeated but it's fine to have an anatomical entity associated with both a region and a layer.